Compat template

ZestyTS opened the pull request:
Made the compat page into a template.

NicoAICP commented:
@ZestyTS I updated the Template to display, as example, NDS instead of NDSCompat.

There is 2 Issues i found out tho:

- in the src/router/index.js files, your props do nothing. Thats why i made aboves commit.

- The Navigation bar does not work anymore. Open /nescompat for example via the url and then press the Compatibility -> NDS button. It will stay at NES. Only way to switch is NES -> HOME -> NDS

ZestyTS commented:
@NicoAICP I made some changes.

- Fixed routing, now you can go directly to the other page without having to go back to home
- Removed the props and instead made a dynamic route
- Fixed the code for getting the json.

There is currently a bug and that has to do with manually typing in the URL, I believe that when it goes live that bug will fix itself since gh-pages doesn't do dynamic routing in the first place.

NicoAICP commented:
Looks good.
